.ci/pytorch/test.sh

```diff
@@ -1404,6 +1404,8 @@ if ! [[ "${BUILD_ENVIRONMENT}" == *libtorch* || "${BUILD_ENVIRONMENT}" == *-baze
   (cd test && python -c "import torch; print(torch.__config__.show())")
   (cd test && python -c "import torch; print(torch.__config__.parallel_info())")
 fi
+
+pip_install --user "fbscribelogger"
```

Comment:
please add this one to https://github.com/pytorch/pytorch/blob/main/.ci/docker/requirements-ci.txt otherwise it will likely to fail

Reply:
yeah I did it more "properly" in https://github.com/pytorch/pytorch/pull/135224